salt/modules/vagrant.py

```diff
@@ -475,20 +479,26 @@ def pause(name):
     return ret == 0
 
 
-def reboot(name):
+def reboot(name, **kwargs):
     '''
     Reboot a VM. (vagrant reload)
 
+    keyword argument:
+
+    - provision: (False) also re-run the Vagrant provisioning scripts.
+
     CLI Example:
 
     .. code-block:: bash
 
-        salt <host> vagrant.reboot <salt_id>
+        salt <host> vagrant.reboot <salt_id> provision=True
     '''
     vm_ = get_vm_info(name)
     machine = vm_['machine']
+    prov =  kwargs.get('provision', False)
```

Comment:
If this is the only keyword argument expected it feels like it would be better just to set `provision=False` in the function interface.

Reply:
Doh!  \<slaps self on head\> Thank you.